third_party/zlib/google/BUILD.gn

```diff
@@ -4,7 +4,7 @@
 
 import("//build_overrides/build.gni")
 
-if (build_with_chromium) {
+if (build_with_chromium || use_cobalt_customizations) {
   static_library("zip") {
```

Comment:
Is something in metrics_proto depending on this target?

Reply:
components/metrics actually: http://cs/cobalt/components/metrics/BUILD.gn;l=118;rcl=9fa886bd2614ad72042cd2565b93673f3483aa17

Reply:
Sg—generally I like to enable targets at the same time as adding an explicit dependency somewhere to make it clear what's building it, but esp with 3P libraries it can be annoying and isn't a big deal